### PR TITLE
Modified model_path variable for .joblib files

### PR DESCRIPTION
**Pull request by jpathak-CS**
Issue for the PR: https://github.com/CognitiveScale/certifai/issues/3631

### CODE REVIEW

**Comment by jpathak-CS** [comment]
Logs after modifications:

```
Tips: You can try get_logs(): https://aka.ms/debugimage#dockerlog or local deployment: https://aka.ms/debugimage#debug-locally to debug if deployment takes longer than 10 minutes.
Running
2021-07-09 13:34:49+00:00 Creating Container Registry if not exists.
2021-07-09 13:34:49+00:00 Registering the environment.
2021-07-09 13:34:49+00:00 Use the existing image.
2021-07-09 13:34:49+00:00 Generating deployment configuration.
2021-07-09 13:34:50+00:00 Submitting deployment to compute.
2021-07-09 13:34:52+00:00 Checking the status of deployment german-credit-logistic-service..
2021-07-09 13:35:03+00:00 Checking the status of inference endpoint german-credit-logistic-service.
Succeeded
ACI service creation operation finished, operation "Succeeded"
Tips: You can try get_logs(): https://aka.ms/debugimage#dockerlog or local deployment: https://aka.ms/debugimage#debug-locally to debug if deployment takes longer than 10 minutes.
SucceededACI service creation operation finished, operation "Succeeded"
```

**Review by laguirre-cs — APPROVED** [review]
Changes look fine to me, and my sole comment is just an observation

File: notebooks/azureml_model_headers_demo/scripts/logistic_score.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ def init():
     # It is the path to the model folder (./azureml-models/$MODEL_NAME/$VERSION)
     # For multiple models, it points to the folder containing all deployed models (./azureml-models)
     model_dir = os.getenv("AZUREML_MODEL_DIR")
-    print(os.listdir(model_dir))
-    model_path = os.path.join(model_dir, f'{model_dir.split("/")[1]}.joblib')
+    # add the model_name.joblib file to the model_path
+    model_path = os.path.join(model_dir, os.listdir(model_dir)[0])
```
[the remark this answers is not in the document]
Comment (jpathak-CS) [reply notebooks/azureml_model_headers_demo/scripts/logistic_score.py:20]: @laguirre-cs String split using `/` was leading to wrong file name, which did not existed. My approach was to print the content within the directory where `model_name.joblib` existed and thus redirected the `model_path` variable to the correct path.

[the remark this answers is not in the document]
Comment (cdraper-CS) [reply notebooks/azureml_model_headers_demo/scripts/logistic_score.py:20]: I think this only works in the first case when the env var points to the single model folder. 
https://docs.microsoft.com/en-us/azure/machine-learning/how-to-deploy-advanced-entry-script#load-registered-models
Perhaps just make it clear in the comment and perhaps refer to that link

**Review by cdraper-CS — APPROVED** [review]

File: notebooks/azureml_model_headers_demo/scripts/logistic_score.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpathak-CS) [reply notebooks/azureml_model_headers_demo/scripts/logistic_score.py:23]: Added Link for `AZURE_ML_MODEL` directory, for enhancement in understanding.

**Pull request merged**